src/jSignature.js

```diff
@@ -1357,8 +1357,11 @@ var GlobalJSignatureObjectInitializer = function(window){
 						sig.$controlbarUpper.remove();
 						$(sig.canvas).remove();
 						var globalEvents = $.fn[apinamespace]('globalEvents');
```

Comment:
Just realised that globalEvents should be available and this line might be redundant. Can you verify and remove?